dali/operators/reader/loader/video_loader.h

```diff
@@ -73,7 +74,7 @@ std::vector<dali::file_meta> get_file_label_pair(const std::string& path,
 }  // namespace filesystem
 
 struct OpenFile {
-  bool open = false;
+  bool loaded = false;
```

Comment:
I don't like this new name - and I think this field is not really necessary.

Reply:
Done